torch_ort/experimental/graph_config.py

```diff
@@ -0,0 +1,56 @@
+from onnxruntime.capi._pybind_state import PropagateCastOpsStrategy
+from torch_ort import ORTModule
+
+
+def set_propagate_cast_ops_optimization(model: ORTModule,
+                                        strategy : PropagateCastOpsStrategy=PropagateCastOpsStrategy.NONE,
+                                        level: int=-1) -> None:
```

Comment:
Instead of int level, may we use an ```IntEnum```? We should have probably introduced this at the ```ORTModule``` level?